tests/test_Dataset.py

```diff
@@ -905,8 +905,10 @@ def __getitem__(self, item):
 def test_DistributeFilesDataset_distribute_evenly_by_size():
     from returnn.datasets.distrib_files import DistributeFilesDataset
 
-    def _test(sizes: List[int], partition_epoch: int, expected: List[List[int]]):
-        files = [f"file-{i}" for i in range(len(sizes))]
+    def _test(sizes: List[int], partition_epoch: int, expected: List[List[int]], files_order=None):
```

Comment:
Oh another small nit (probably requires reformatting w/ black):
```suggestion
    def _test(sizes: List[int], partition_epoch: int, expected: List[List[int]], files_order: Optional[List[str]] = None):
```